### PR TITLE
documentation pages

### PR DESCRIPTION
**Pull request by notwaldorf**
suuuuuper work in progress but also sort of ready for review

### CODE REVIEW

File: docs/_layouts/post.html [hunk 1]
```diff
@@ -0,0 +1,77 @@
+---
+layout: default
+---
+<link rel="stylesheet" href="{{site.baseurl}}/css/pygments.css">
+<main>
+  <nav class="toc">
+    <ul data-depth="1">
+
+      <li data-level="H2">
```
Comment (notwaldorf) [inline docs/_layouts/post.html]: not sure whether this data-level is needed anymore

File: docs/_layouts/default.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+  <footer>
+    <p>© 2018 Polymer Authors. Code licensed under the
+    <a target="_blank" href="http://polymer.github.io/LICENSE.txt"> BSD License</a>. </p>
+
+    <ul>
+      <li><a href="https://www.github.com/polymer/pwa-starter-kit" rel="noopener" target="_blank">GitHub</a></li>
+      <li><a href="https://www.twitter.com/polymer" rel="noopener" target="_blank">Twitter</a></li>
+      <li><a href="https://www.youtube.com/watch?v=we3lLo-UFtk" rel="noopener" target="_blank">YouTube</a></li>
+
```
Comment (frankiefu) [inline docs/_layouts/default.html]: Missing the closing `</ul>` here.

File: docs/_layouts/post.html (hunk 1 above)
Comment (frankiefu) [reply docs/_layouts/post.html]: Yeah I think it's not needed anymore. Also remove `data-depth` too.

File: docs/_posts/2018-05-31-Setup.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+This will start a local server on port `8081`. Open [http://localhost:8081](http://localhost:8081) to view your app in the browser. Note that this server can continue running as you're making changes to your application, which you will see if you refresh the browser tab.
+
+If the port is already taken on your computer, or if you need to change the default hostname (because you're using a Docker container, for example), you can configure them using command line arguments:
+```
+npm start -- --hostname 0.0.0.0 --port 4444
+```
+
+### Run the tests
+Check out the [Application testing](https://github.com/Polymer/pwa-starter-kit/wiki/7.-Application-testing) page for more information about the tests. For a quick way to run the tests, run
```
Comment (frankiefu) [inline docs/_posts/2018-05-31-Setup.md]: Need to update all the links to other pages to relative paths, e.g.
`https://github.com/Polymer/pwa-starter-kit/wiki/7.-Application-testing`
to 
`/Application-testing`

File: docs/_layouts/default.html [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
+  <meta name="description" content="An end-to-end solution for building high-quality PWAs">
+  <title>{% if page.title %}{{ page.title }} &ndash;{% endif %} {{ site.name }}</title>
+  <link rel="shortcut icon" sizes="32x32" href="{{site.baseurl}}/images/icon-32.png">
+  <meta name="theme-color" content="#000">
```
Comment (frankiefu) [inline docs/_layouts/default.html]: I think this is another good use-case of preconnect:
```
<link rel="preconnect" href="https://fonts.gstatic.com/" crossorigin>
```

File: docs/css/main.css [hunk 5]
```diff
@@ -0,0 +1,521 @@
...
+
+code, pre {
+  font-family: 'Source Code Pro', 'Roboto Mono', Consolas, Menlo, monospace !important;
+}
+
+code {
+  background: #F4F4F6;
+  color: black;
+  font-size: inherit;
+  padding: 4px 8px;
```
Comment (frankiefu) [inline docs/css/main.css]: The padding makes the code block looks weird. Can we remove it?
screen shot 2018-06-12 at 4 34 24 pm

**Comment by notwaldorf** [comment]
addressed the review comments; going to close this pr, and make a new gh-pages branch as discussed.

**Pull request closed without merging**